yesod-newsfeed/Yesod/Common.hs

```diff
@@ -0,0 +1,9 @@
+module Yesod.Common
+  ( removeItem
```

Comment:
it isn't used anymore. I used it to clean the lists up a bit, which didn't work out.